policy/policy.go

```diff
@@ -110,8 +110,10 @@ func WaitUntilDone(resolver PolicyResolver, entity string, scoringMrn string, ti
 
 // RefreshLocalAssetFilters looks through the local policy asset filters and rolls them up
 func (p *Policy) RefreshLocalAssetFilters() {
-	p.Filters = &explorer.Filters{
-		Items: map[string]*explorer.Mquery{},
+	if p.Filters == nil {
+		p.Filters = &explorer.Filters{
+			Items: map[string]*explorer.Mquery{},
```

Comment:
I think we should separately check if items is nil

Reply:
Added the check